torchvision/prototype/transforms/_utils.py

```diff
@@ -31,12 +33,10 @@ def _check_fill_arg(fill: Optional[Union[FillType, Dict[Type, FillType]]]) -> No
             raise TypeError("Got inappropriate fill arg")
 
 
-def _setup_fill_arg(
-    fill: Optional[Union[FillType, Dict[Type, FillType]]]
-) -> Union[Dict[Type, FillType], Dict[Type, None]]:
+def _setup_fill_arg(fill: Union[FillType, Dict[Type, FillType]]) -> Optional[Dict[Type, FillType]]:
     _check_fill_arg(fill)
 
-    if isinstance(fill, dict):
+    if fill is None or isinstance(fill, dict):
```

Comment:
I allow for fill to be directly `None` instead of forcing it to be a dictionary that returns `None` as this simplifies the type declarations everywhere. This is aligned with an earlier proposal for @vfdev-5